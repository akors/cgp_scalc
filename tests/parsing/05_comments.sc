# this is a line starting with a comment
22 - 1 # comment after text

# comment after empty line
1 this should give an error 2 #comment after error
17 - 1 # comment after # comment after # comment
# coµµent wiß wiürd charäcters
# comment at the end of file