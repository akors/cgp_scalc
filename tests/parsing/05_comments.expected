21
syntax error, unexpected IDENTIFIER. line 5
16
